content/v2.0/cloud/account-management/billing.md

```diff
@@ -10,25 +10,27 @@ menu:
     name: Add payment and view billing
 ---
 
-- Hover over the **Usage** icon in the left navigation bar and select **Billing**.
+Add your payment method and view billing information in the {{< cloud-name "short" >}} user interface (UI).
 
-    {{< nav-icon "cloud" >}}
+## Billing details
+To access billing information, click on the **user avatar** in the top right corner
```

Comment:
1. Click the **user....
2. Do one of the following:
    - If you....through AWS...
    - If you...through InfluxData...